add link to DeepField

DeepFieldAdmin opened the pull request:
Add https://github.com/deepfield-team/DeepField to the list of reservoir engineering tools

<!-- Thank you for contributing to our list! -->
<!-- Please write a DESCRIPTIVE TITLE for the pull request and commits. -->


__Project name:__ DeepField

__Project website/repository:__https://github.com/deepfield-team/DeepField

__License:__MIT

__Submission type:__
<!-- Submission type can be one of "New Software Project", "New Data Repository", "New Tutorial or Cheat Sheet", "New Miscellaneous", or "Fix/update existing entry" -->New Software Project


<!-- In the following checklist, an empty check-box is "[ ]", a checked check-box is "[x]". You can also interactively click the boxes once the PR is submitted. -->

## Checklist for all pull requests:
<!-- If adding a project to the list, make sure it fulfills the following criteria. -->

<!-- Make sure it's "certified awesome"! -->
- [ ] Project is [`certified awesome`](awesome.md)

<!-- General requirements -->
- [x] The project is [open-source](https://opensource.org/licenses/alphabetical) and accessible.
- [x] Searched the existing entries to make sure this is not a duplicate.
- [x] Contains only a single addition (make separate PRs if adding more than one).
- [x] Added the link to the bottom of the relevant category.
- [x] Created a new category only if necessary.

<!-- Formatting criteria -->
- [x] Add icon from the `media/icon/` folder if applicable (e.g., `![Python](media/icon/python.png)` ).
- [x] Checked spelling and grammar.
- [x] Removed trailing whitespace and periods.
- [x] Confirm the dash – is not a minus -.
- [x] Used [title-casing](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case) (AP style) for project name.


### Checklist for new software projects:

- [x] Link the code repository rather than the website/documentation
- [x] Installation instructions present
- [x] Documentation looks great
- [x] Examples or Tutorials to follow
- [x] Tests and Travis CI running


### Checklist for new Data Repositories, Tutorials and Cheat Sheets, and Miscellaneous:

- [x] Easily accessible
- [x] Comprehensive and widely appealing

JustinGOSSES commented:
Interesting project. API is documented well but would benefit from a "getting started" or "tutorial" page that runs through using steps of how to use Deep Field with one of the open data models I see are included. This would help people who aren't already deep into the code figure out how to use it.

Additionally, you might consider adding a CONTRIBUTING.md to make clear whether this project accepts contributions from outside the code group and how best to do that. https://github.com/deepfield-team/DeepField/community 

There is an open data folder but no mention of open source  licenses associated with the data that I saw. I googled one of them dataset and it seems providing it to others in this manner may not be in line with the license: http://www.ipt.ntnu.no/~norne/wiki1/doku.php . It is not clear.

DeepFieldAdmin commented:
Hi, @JustinGOSSES! Thanks for the good suggestions. We added "Quick start" section to the documentation, added CONTRIBUTING.md, and described terms of use of the models provided in the open_data directory. Is everything fine now to complete the pull request?

JustinGOSSES commented:
Thanks @DeepFieldAdmin , great improvements!
